ormin/ormin_sqlite.nim

```diff
@@ -34,7 +34,7 @@ template bindParam*(db: DbConn; s: PStmt; idx: int; x, t: untyped) =
   when t is int or t is int64 or t is bool:
     if bind_int64(s, idx.cint, x.int64) != SQLITE_OK: dbError(db)
   elif t is string:
-    if bind_blob(s, idx.cint, cstring(x), x.len.cint, SQLITE_STATIC) != SQLITE_OK:
+    if bind_text(s, idx.cint, cstring(x), x.len.cint, SQLITE_STATIC) != SQLITE_OK:
```

Comment:
From https://www.sqlite.org/datatype3.html

TEXT. The value is a text string, stored using the database encoding (UTF-8, UTF-16BE or UTF-16LE).

BLOB. The value is a blob of data, stored exactly as it was input.


So this means there are some compat problems. Maybe we can determine the type from the schema information?

Reply:
Since the schema is defined as the text type, we should also be consistent when binding parameters, we should use bind_text()，so that query conditions like this: `where typstring == "one"`  or `where typstring =~ "%on%"` and string functions can work correctly.

Reply:
Blob types are handled separately and can easily store binary objects directly.